packages/carousel/src/Default/stories/Default-index.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { component as Carousel } from '../index';
+import { CarouselDefaultProps, Slide } from '../Default.types';
+
+import { appearance } from './Default-appearance.stories';
+import { size } from './Default-size.stories';
+import { theme } from './Default-theme.stories';
+
+const slides: Slide[] = [
+  {
+    id: '1',
+    style: { backgroundColor: '#017aba' },
+    backgroundContent: '',
+    caption: (
+      <>
+        <h3>Slide 1 label</h3>
+        <p>Slide 1 caption text. Text example.</p>
+      </>
+    ),
+    bsItemProps: {
+      interval: 3000,
+    },
+  },
+  {
+    id: '2',
+    style: { backgroundColor: '#6a98b0' },
+    backgroundContent: '',
+    caption: (
+      <>
+        <h3>Slide 2 label</h3>
+        <p>Slide 2 caption text. Text example.</p>
+      </>
+    ),
+    bsItemProps: {
+      interval: 6000,
+    },
+    bsCaptionProps: {
+      as: 'li',
+      bsPrefix: 'example-caption-prefix owlui-carousel-caption',
```

Comment:
Noticed we are rendering an li element as a child of a div - are we okay to do this? Or would we want to render Carousel item as a ul element instead for W3 purposes?

Reply:
Never okay, but this is just an example of how to use the `as` prop that will be rendered on Storybook only, so it's fine to keep it.